charts/zeebe-operate-helm/templates/service.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: {{ include "zeebe-operate.fullname" . }}
   labels:
     app: {{ include "zeebe-operate.fullname" . }}
+    {{- toYaml  .Values.labels | nindent 4 }}
```

Comment:
Is this really what we/you want if you specify a label that the service and the deployment has the same deployment? :thinking: currently not 100% sure.

Reply:
Lets do it when we see a need for it